.tx/config

```diff
@@ -9,31 +9,34 @@ source_lang = en
 type = KEYVALUEJSON
 
 # Mappings from Transifex locale codes to Valhalla file names
-trans.bg = locales/bg-BG.json
+trans.ar_SA = locales/ar-SA.json
+trans.bg_BG = locales/bg-BG.json
 trans.ca_ES = locales/ca-ES.json
-trans.cs = locales/cs-CZ.json
```

Comment:
I regionalized all transifex translations to be unambiguous which locale it's referring to